src/DependencyInjection/SonataAdminExtension.php

```diff
@@ -226,6 +226,8 @@ public function getNamespace()
 
     private function buildStylesheets(array $config): array
     {
+        $config['assets']['stylesheets'][] = "bundles/sonataadmin/vendor/admin-lte/dist/css/skins/{$config['options']['skin']}.min.css";
```

Comment:
Is there any requirement to use [complex variable parsing syntax](https://www.php.net/manual/en/language.types.string.php#language.types.string.parsing.complex) here? We usually use `sprintf()` instead.